### DISCUSSION TITLE
Migrating Database to Saleor Instance

### DISCUSSION THREAD

**Discussion by ishanInstaCash** [saleor/saleor#16741] [Q&A]
I have a question, i had made an application where i was using an SQL database now i want to migrate this database to Saleor Instance is there any possible way i can achieve this ?

**Comment by tomaszlud** [comment]
Currently, there's no way to upload a custom database to Saleor so that it automatically maps everything. The simplest solution is to set up a sandbox with an empty database and load whatever you need into Saleor via the API.
